terraform/aws/app-infrastructure/aws-prometheus-grafana/modules/prometheus-helm/values.yaml

```diff
@@ -12,10 +12,9 @@ imagePullSecrets: []
 serviceAccounts:
   server:
     create: true
-    name: ${SERVICE_ACCOUNT_NAME}
+    name: "cdc-nbs-fts2-amp-svc-acc"  # ${SERVICE_ACCOUNT_NAME}
     annotations: 
-      eks.amazonaws.com/role-arn: ${IAM_PROXY_PROMETHEUS_ROLE_ARN}
-
+      eks.amazonaws.com/role-arn: "arn:aws:iam::324406605642:role/cdc-nbs-fts2-amp-role"  # ${IAM_PROXY_PROMETHEUS_ROLE_ARN}
```

Comment:
there is an account id, this cannot be part of PR or commit history really. please check how the role is passed in.